[DOCS+] Add #158940 to the list of 8.8.0 known issues (#159197)
elastic/kibana: docs/CHANGELOG.asciidoc

====
// end::known-issue-156798[]

// tag::known-issue-158940[]
[discrete]

> .Failed upgrades to 8.8.0 could cause bootlooping and possibly data loss during upgrade

Reviewer: ```suggestion
.Failed upgrades to 8.8.0 can cause bootlooping and data loss
[%collapsible]
```
